core/services/workflows/metering/metering.go

```diff
@@ -541,3 +559,35 @@ func (s *Reports) Len() int {
 
 	return len(s.reports)
 }
+
+func RatiosFromConfig(
+	info capabilities.CapabilityInfo,
+	config *values.Map,
+) (map[capabilities.CapabilitySpendType]decimal.Decimal, error) {
+	ratios := make(map[capabilities.CapabilitySpendType]decimal.Decimal)
+
+	for _, spendType := range info.SpendTypes {
+		// using a namespace on the config key to distinguish billing specific keys
+		key := fmt.Sprintf("billing_%s", spendType)
```

Comment:
Could we just use a nested map instead?
Besides polluting the outermost config it also makes it hard for other people to know what these are.
If it was a submap keyed under something like `spendRatios` it would make it less arcane.